internal/seclang/rules_test.go

```diff
@@ -239,12 +239,81 @@ func TestTagsAreNotPrintedTwice(t *testing.T) {
 	}
 	re := regexp.MustCompile(`\[tag "some1"\]`)
 	for _, l := range logs {
-		if len(re.FindAllString(l, -1)) < 2 {
```

Comment:
I don't fully get this test. The title (`TestTagsAreNotPrintedTwice`) seems like that wants to avoid the same tag appearing more than once inside the logs, but then we are trying to find the same tag twice. Originally it was enforcing being just one, but it has been changed with https://github.com/corazawaf/coraza/pull/220/files#diff-090979aa58246451224457e41b61bf533048a2e7091724b5db0345c04f5ef561L248-R249.
Currently, the test is trying to match the following: 
```
"[client \"\"] Coraza: Warning.  [file \"\"] [line \"2\"] [id \"1\"] [rev \"\"] [msg \"\"] [data \"\"] [severity \"emergency\"] [ver \"\"] [maturity \"0\"] [accuracy \"0\"] [tag \"some1\"] [tag \"some2\"] [hostname \"\"] [uri \"\"] [unique_id \"xDFbZiZkbAmlAAKUNXc\"]\n[client \"\"] Coraza: Warning.  [file \"\"] [line \"2\"] [id \"1\"] [rev \"\"] [msg \"\"] [data \"\"] [severity \"emergency\"] [ver \"\"] [maturity \"0\"] [accuracy \"0\"] [tag \"some1\"] [tag \"some2\"] [hostname \"\"] [uri \"\"] [unique_id \"xDFbZiZkbAmlAAKUNXc\"]\n"
```
And it seems that the following part is very lengthy and not relevant (Current PR tries to avoid redundancy, therefore it does not print it):
```
[client \"\"] Coraza: Warning.  [file \"\"] [line \"2\"] [id \"1\"] [rev \"\"] [msg \"\"] [data \"\"] [severity \"emergency\"] [ver \"\"] [maturity \"0\"] [accuracy \"0\"] [tag \"some1\"] [tag \"some2\"] [hostname \"\"] [uri \"\"] [unique_id \"xDFbZiZkbAmlAAKUNXc\"]\n"
```
cc. @piyushroshan

Reply:
Ping @jptosso